astropy/tests/plugins/display.py

```diff
@@ -96,10 +96,14 @@ def pytest_report_header(config):
 
     # Helpers version
     try:
-        from ...version import astropy_helpers_version
-    except ImportError:
-        pass
-    else:
+        astropy_helpers_version = TESTED_VERSIONS['astropy_helpers']
+    except KeyError:
```

Comment:
So this `except KeyError` block is going to get hit a lot by Astropy core itself and other packages that use this `display` plugin but did not set `TESTED_VERSIONS['astropy_helpers']`. Given that `except` is "expernsive", might want to consider change this `try ... except ...` to `if ... else ...`? This is in no way a bottleneck, so maybe my suggestion falls in the premature optimization category. 🤷‍♀️  Just putting my thought here for your consideration.

Reply:
I feel it shouldn't matter given this only runs once and test collection is slow anyway.

But I see no reason to not do it with an if/else instead.